src/components/PreviewDraft/PreviewConceptComponent.tsx

```diff
@@ -61,16 +61,18 @@ const PreviewConceptComponent = ({ concept, language }: Props) => {
   }
 
   return (
-    <ConceptNotionV2
-      title={concept.title}
-      content={parsedContent}
+    <BlockConcept
+      title={concept.title.title}
       visualElement={visualElementMeta}
       copyright={concept.copyright}
-      tags={concept.tags?.tags}
-      subjects={subjects?.results?.map((res) => res.name)}
-      conceptType={concept.conceptType}
+      // TODO: Consider if we should add support for these again
+      // tags={concept.tags?.tags}
+      // subjects={subjects?.results?.map((res) => res.name)}
+      // conceptType={concept.conceptType}
```

Comment:
Brukes kun i liste så tenker vi kan ta det bort herfra.

Reply:
I og med at liste ryker, betyr det at vi kan droppe støtte for det overalt?

Reply:
På sikt, sikkert.